aws/resource_aws_elb.go

```diff
@@ -44,6 +45,12 @@ func resourceAwsElb() *schema.Resource {
 				ValidateFunc: validateElbNamePrefix,
 			},
 
+			"arn": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
```

Comment:
Any particular reason why it should be optional? i.e. why would the user need to define it in the config?

Reply:
Ah sorry, delicious copypasta. It should probably always be just Computed.